addons/web/static/src/views/graph/graph_view.js

```diff
@@ -43,6 +43,7 @@ export const graphView = {
                 order: archInfo.order || null,
                 resModel: resModel,
                 stacked: "stacked" in archInfo ? archInfo.stacked : true,
+                cumulated: "cumulated" in archInfo ? archInfo.cumulated : false,
```

Comment:
Two shorter way to do the same thing
!!archInfo.cumulated
or
Boolean(archInfo.cumulated)